packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -88,15 +90,21 @@ const Footer = props => {
                             defaultMessage={LOREM_IPSUM}
                         />
                     </p>
-                    <ul className={classes.socialLinks}>
+                    <ul className='footer-socialLinks'>
                         <li>
-                            <Instagram size={20} />
+                            <FontAwesomeIcon icon={faBehance} className='footer-socialIcon'/>
                         </li>
                         <li>
-                            <Facebook size={20} />
+                            <FontAwesomeIcon icon={faFacebook} className='footer-socialIcon'/>
                         </li>
                         <li>
-                            <Twitter size={20} />
+                            <FontAwesomeIcon icon={faGoogle} className='footer-socialIcon'/>
+                        </li>
+                        <li>
+                            <FontAwesomeIcon icon={faSkype} className='footer-socialIcon'/>
```

Comment:
Try not to use string values as classNames
All styles must be imported from classes object